terraform/application/variables.tf

```
@@ -17,6 +17,8 @@ variable "azure_resource_prefix" {
   type = string
 }
 
+variable "azure_maintenance_window" { default = null }
```

```suggestion
variable "azure_maintenance_window" {
  type = object({
      day_of_week  = number
      start_hour   = number
      start_minute = number
  })
  default = null
}
```

Just to make sure we always pass this in correctly.

---

ya it make sense , updated the codes.